internal/server/v1/event/golden/event_collection_test.go

```diff
@@ -74,6 +74,16 @@ func TestEventCollection(t *testing.T) {
 				8,
 				"FireEvent_CA_202210.json",
 			},
+			{
+				"WildlandFireEvent",
+				"Earth",
+				"2022-10",
```

Comment:
The added test file has 100K lines. As this is only for testing purpose, can we make the filter here even more like (7,8) to reduce the size?

Reply:
done - updated filter & changed from Earth to California to cut down test file size